modules/meb_api/app/controllers/meb_api/v0/base_controller.rb

```diff
@@ -5,6 +5,7 @@ module V0
     class BaseController < ::ApplicationController
       before_action :check_flipper
       skip_before_action :authenticate
+      #       skip_before_action :verify_authenticity_token
```

Comment:
Is this meant to be commented out?

Reply:
@LindseySaari : Yes, but I've taken out the comment now on new push. to address your comment. Thanks.

Reply:
Ok great! Sounds good, it looks like you'll have to update this branch with the lastest master. There was a fix to bump the brakeman version.

Reply:
I've just merged master into branch and pushed.